test/jdk/javax/swing/JTextPane/8180276/ChangeStyleAndAppend.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Dialog;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.Panel;
+import java.awt.TextArea;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import javax.swing.border.EtchedBorder;
+import javax.swing.JFrame;
+import javax.swing.JTextPane;
+import javax.swing.SwingUtilities;
+import javax.swing.text.BadLocationException;
+import javax.swing.text.Document;
+
+/*
+ * @test
+ * @bug 8180276
+ * @summary Verifies if getText method returns extra carriage return when
+ *          mixed with methods of document.
+ * @key headful
+ * @run main/manual ChangeStyleAndAppend
+ */
+
+public class ChangeStyleAndAppend extends JFrame {
+    static Dialog controlDialog;
+    private AtomicReference<Boolean> testResult = new AtomicReference<>(false);
+    static final CountDownLatch latch = new CountDownLatch(1);
+    class MyJTextPane extends JTextPane {
+        public void appendLine(String s) {
+            try {
+                Document doc = this.getDocument();
+                doc.insertString(doc.getLength(), s + System.lineSeparator(), null);
+            } catch(BadLocationException e) {
```

Comment:
```suggestion
            } catch (BadLocationException e) {
```
The should be space between `catch` keyword and the opening parenthesis.